presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1044,6 +1046,30 @@ public FeaturesConfig setPartialAggregationByteReductionThreshold(double partial
         return this;
     }
 
+    public boolean isAdaptivePartialAggregationEnabled()
+    {
+        return adaptivePartialAggregationEnabled;
+    }
+
+    @Config("experimental.adaptive-partial-aggregation")
+    public FeaturesConfig setAdaptivePartialAggregationEnabled(boolean adaptivePartialAggregationEnabled)
+    {
+        this.adaptivePartialAggregationEnabled = adaptivePartialAggregationEnabled;
+        return this;
+    }
+
+    public double getAdaptivePartialAggregationRowsReductionRatioThreshold()
+    {
+        return adaptivePartialAggregationRowsReductionRatioThreshold;
+    }
+
+    @Config("experimental.adaptive-partial-aggregation-rows-reduction-ratio-threshold")
```

Comment:
Q: What is the policy b/w the feature config name and the session prop name agreeing on the prefix experimental ?

Reply:
There is no policy - session prop name is totally different from feature config name and session props do not have prefixes unto themselves. When they do have prefix, it's to refer to the catalog to which the session prop should be applied.